src/Resources/skeleton/doctrine/Entity.tpl.php

```diff
@@ -26,7 +23,7 @@ class <?= $class_name."\n" ?>
     /**
      * @return int|null
      */
-    public function getId(): ?int
```

Comment:
It's php `7.0.8` defined in composer.json
But return type `or null` come just in php 7.1

Reply:
make:entity [requires PHP 7.1](https://github.com/symfony/maker-bundle/blob/master/src/Maker/MakeEntity.php#L280)

Reply:
That's correct. Let's revert this change.